### PR TITLE
Add log in event hub to help troubleshooting

### PR DESCRIPTION
**Pull request by warrenzhu25**

### CODE REVIEW

File: spring-integration-azure/spring-integration-azure-test/src/main/java/com/microsoft/azure/spring/integration/test/support/MessageHandlerTest.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
     protected DefaultMessageHandler handler;
     protected String destination = "dest";
+    protected String dynamicDestination = "dynamicName";
```
Comment (Incarnation-p-lee) [inline spring-integration-azure/spring-integration-azure-test/src/main/java/com/microsoft/azure/spring/integration/test/support/MessageHandlerTest.java]: not sure if `dynamicDestination ` can be final or not.

File: spring-integration-azure/spring-integration-eventhub/src/main/java/com/microsoft/azure/spring/integration/eventhub/util/EventDataHelper.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+package com.microsoft.azure.spring.integration.eventhub.util;
+
+import com.microsoft.azure.eventhubs.EventData;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class EventDataHelper {
+
+    public static String toString(EventData eventData){
```
Comment (Incarnation-p-lee) [inline spring-integration-azure/spring-integration-eventhub/src/main/java/com/microsoft/azure/spring/integration/eventhub/util/EventDataHelper.java]: `EventData` is interface, and you would like not to put this `toString()` to interface, as it just for logging, right ?

Comment (warrenzhu25) [reply spring-integration-azure/spring-integration-eventhub/src/main/java/com/microsoft/azure/spring/integration/eventhub/util/EventDataHelper.java]: `EventData` is provided by event hub SDK, we have no control.

File: spring-integration-azure/spring-integration-azure-test/src/main/java/com/microsoft/azure/spring/integration/test/support/MessageHandlerTest.java (hunk 1 above)
Comment (warrenzhu25) [reply spring-integration-azure/spring-integration-azure-test/src/main/java/com/microsoft/azure/spring/integration/test/support/MessageHandlerTest.java]: could be final.

**Pull request merged**